linter/allowlists/default.txt

```diff
@@ -350,3 +350,13 @@ WebP
 xliff
 xls
 yml
+release
+Release
+Releases
+releasing
+stages
+Stages
+auto
+deployment
+Prod
+Stage
```

Comment:
I do not think all of these are necessary. For example `releasing` does not exist in the German title or description, so I don't think it needs to be specifically allowed for all languages?